src/terminal/parser/base64.cpp

```diff
@@ -176,7 +177,11 @@ bool Base64::s_Decode(const std::wstring_view src, std::wstring& dst) noexcept
         return false;
     }
 
-    return true;
+    const auto wstrLen = MultiByteToWideChar(CP_UTF8, 0, mbStr.c_str(), (int)mbStr.length(), nullptr, 0);
+    dst.resize(wstrLen);
+    const auto err = MultiByteToWideChar(CP_UTF8, 0, mbStr.c_str(), (int)mbStr.length(), dst.data(), wstrLen);

```

Comment:
Would you mind switching this to `til::u8u16`? We can safely replace this with ...

```suggestion
    dst = std::move(til::u8u16(mbStr));
```

It wraps up MultiByteToWideChar but also supports collecting partials and deals w/ the double-call interface (by estimating the required length.)

Reply:
you can probably use the HRESULT version actually.

```
return SUCCEEDED(til::u8u16(mbStr, dst));
```

:smile:

Reply:
It works as expected, thanks!